src/main/java/rx/internal/util/PaddedAtomicIntegerBase.java

```diff
@@ -26,55 +26,49 @@
 
     private static final long serialVersionUID = 6513142711280243198L;
 
-    private static final AtomicIntegerFieldUpdater<PaddedAtomicIntegerBase> updater;
-
-    static {
-        updater = AtomicIntegerFieldUpdater.newUpdater(PaddedAtomicIntegerBase.class, "value");
-    }
-
-    private volatile int value; // 8-byte object field (or 4-byte + padding)
+    private AtomicInteger value = new AtomicInteger(); // 8-byte object field (or 4-byte + padding)
 
     public final int get() {
-        return value;
+        return value.get();
     }
 
     public final void set(int newValue) {
-        this.value = newValue;
+        this.value.set(newValue);
     }
 
     public final void lazySet(int newValue) {
-        updater.lazySet(this, newValue);
+        value.lazySet(newValue);
     }
 
     public final boolean compareAndSet(int expect, int update) {
-        return updater.compareAndSet(this, expect, update);
+        return value.compareAndSet(expect, update);
     }
 
     public final boolean weakCompareAndSet(int expect, int update) {
-        return updater.weakCompareAndSet(this, expect, update);
+        return value.weakCompareAndSet(expect, update);
     }
 
     public final int getAndSet(int newValue) {
-        return updater.getAndSet(this, value);
+        return value.getAndSet(newValue);
```

Comment:
This is a change in behavior from before, but it seems to me the old behavior was wrong (setting `value` on `updater` instead of `newValue`)